www/src/pages/insights.jsx

```diff
@@ -44,36 +49,63 @@ const componentsUsage = dependentProjectsUsages.reduce((accumulator, project) =>
   return accumulator;
 }, {});
 
-const summaryComponentsUsage = Object.entries(componentsUsage).map(([componentName, usages]) => {
-  const componentUsageCounts = usages.reduce((accumulator, project) => accumulator += project.componentUsageCount, 0);
-  return {
-    name: componentName,
-    count: componentUsageCounts,
-    usages: componentsUsage[componentName],
-  };
-});
+const round = (n) => Math.round(n * 10) / 10;
+
+const getEmptyMessage = (text) => `Currently there are no ${text} usage yet`;
 
 const SummaryUsage = () => {
+  const { paragonTypes } = useContext(SettingsContext);
+  const isMedium = useMediaQuery({ minWidth: breakpoints.large.minWidth });
+
+  const summaryComponentsUsage = Object.entries(componentsUsage).map(([componentName, usages]) => {
+    const componentUsageCounts = usages.reduce((accumulator, project) => accumulator + project.componentUsageCount, 0);
+    return {
+      name: componentName,
+      count: componentUsageCounts,
+      usages: componentsUsage[componentName],
+      type: paragonTypes[componentName],
+    };
+  });
+
+  const typeCount = dependentProjectsUsages.reduce((accumulator, project) => {
+    Object.keys(project.usages).forEach(componentName => {
+      const type = paragonTypes[componentName];
+      accumulator[type] = (accumulator[type] || 0) + 1;
+    });
+    return accumulator;
+  }, {});
+
+  const filterValues = Object.keys(paragonTypes)
+    .map((key) => paragonTypes[key])
+    .filter((v, i, a) => a.indexOf(v) === i)
+    .map(type => ({ name: type, number: typeCount[type], value: type }));
+
   const summaryTableData = summaryComponentsUsage.sort((a, b) => {
     const nameA = a.name.toUpperCase();
     const nameB = b.name.toUpperCase();
-    return (nameA < nameB) ? -1 : (nameA > nameB) ? 1 : 0;
+    return nameA < nameB ? -1 : 1;
```

Comment:
Is this fundamentally changing the sort logic? It's removing the case where `nameA` is the same as `nameB`, [e.g.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort):
```js
function compare(a, b) {
  if (a is less than b by some ordering criterion) {
    return -1;
  }
  if (a is greater than b by the ordering criterion) {
    return 1;
  }
  // a must be equal to b
  return 0;
}
```

Reply:
`nameA < nameB ? -1 : 1;` will be okay to omit `0` case. Only `-1` case results into values swap, `0` and `1` leave them as they are in the array. Correct me if I am wrong. Also in the table we can not have duplicates.
![image](https://user-images.githubusercontent.com/36768631/175937365-dd460ba3-cac0-45fd-835d-e693812e3e77.png)